src/api/types/fieldsValue.ts

```diff
@@ -0,0 +1,115 @@
+import { RequiredProps } from "../../types";
+import { IssueBean } from "./types";
+import { components } from "./schema";
+
+export type Issuetype = {
+  self: string;
+  id: string;
+  description: string;
+  iconUrl: string;
+  name: string;
+  subtask: boolean;
+  avatarId: number;
+  entityId: string;
+  hierarchyLevel: number;
+};
+
+export type Project = {
+  self: string;
+  id: string;
+  key: string;
+  name: string;
+  projectTypeKey: string; // "software"
```

Comment:
Is this always "software", if so, should it be typed in?

Reply:
This is just an example of one of the real values. I deleted it.